Lib/test/test_descr.py

```diff
@@ -4452,6 +4450,8 @@ def __getattr__(self, attr):
             print("Oops!")
         except RuntimeError:
             pass
+        else:
```

Comment:
Would you like to update the test by using
`from contextlib import redirect_stdout`?

Reply:
Thanks for the idea! But, I went for the smallest diff.